db_scripts/createIdaNewDB.sh

```
@@ -8,32 +8,32 @@ if [[ $# -ne 3 ]]; then
 
 ## create new DB directory
 rm -rf $newDB
 cp -r $oldDB $newDB
 
 ## update DB
-sed -i "s/$oldDB\>/$newDB/g" $newDB/mosip_ida_deploy.properties;
-sed -i "s/$oldDB/$newDB/g"   $newDB/mosip_ida_db.sql;
-sed -i "s/$oldDB/$newDB/g"   $newDB/mosip_ida_ddl_deploy.sql;
-sed -i "s/$oldDB/$newDB/g"   $newDB/mosip_ida_dml_deploy.sql;
-sed -i "s/$oldDB/$newDB/g"   $newDB/mosip_ida_grants.sql;
+sed -i "s/$oldDB\>/$newDB/g" $newDB/:mosipdbname_deploy.properties;
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)